### DISCUSSION TITLE
displayImage mit Geräte-DP  anzeigen

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1996] [Q&A]
Wenn man in dem DP einen base64 hat klappt es.
Wenn man aber in der Anzeigenvariante unterschiedliche Bilder als base64-Strings setzt klappt es nicht?
`{"1":"data:image/png;base64,....","2":"data:image/png;base64,..."}`
Sollte doch eigentlich funktionieren?

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/issues/1999
